astronomer/providers/amazon/aws/sensors/emr.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Dict
+
+from airflow import AirflowException
+from airflow.providers.amazon.aws.sensors.emr import EmrContainerSensor
+
+from astronomer.providers.amazon.aws.triggers.emr import EmrContainerSensorTrigger
+
+
+class EmrContainerSensorAsync(EmrContainerSensor):
+    """
+    EmrContainerSensorAsync is async version of EmrContainerSensor,
+    Asks for the state of the job run until it reaches a failure state or success state.
+    If the job run fails, the task will fail.
+
+    :param virtual_cluster_id: Reference Emr cluster id
+    :param job_id: job_id to check the state
+    :param max_retries: Number of times to poll for query state before
+        returning the current state, defaults to None
+    :param aws_conn_id: aws connection to use, defaults to 'aws_default'
```

Comment:
```suggestion
    :param aws_conn_id: aws connection to use, defaults to ``aws_default``
```

So they are rendered nicely as docstrings use RST

Reply:
FIxed` :param aws_conn_id: aws connection to use, defaults to ``aws_default```